apis_core/apis_entities/management/commands/normalize_uris.py

```diff
@@ -22,10 +16,11 @@ class Command(BaseCommand):
     def handle(self, *args, **options):
         for x in Uri.objects.all():
```

Comment:
I think this whole block should be wrapped in `with transaction.atomic():` that way URIs get only changed if the whole set of changes succeeds.